app/validation/validator.py

```diff
@@ -302,25 +302,29 @@ def validate_when_rule(when_clause, answer_ids_with_group_id, referenced_id):
         Validates any answer id in a when clause exists within the schema
         Will also check that comparison_id exists
         """
-        answer_reference_id_fields = ('id', 'answer_count', 'comparison_id')
+        answer_reference_id_fields = ('id', 'answer_ids', 'answer_count', 'comparison_id')
```

Comment:
Isn't `answer_count` no longer a property that can exist in a when rule? (as it's now a value of a `type` property)